Model/Events.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Klaviyo\Reclaim\Model;
+
+use Magento\Framework\Model\AbstractModel;
+
+/**
+ * Klaviyo Events Model is used to represent the kl_events table created using db_schema.xml file using declarative schema.
+ * https://devdocs.magento.com/guides/v2.4/extension-dev-guide/declarative-schema/
+ * The Model doesn't have the capability to query the DB directly but is required by the ResourceModel to create Magento DataObject instances
```

Comment:
@smoucka This is where I've explained how the model and resourceModel are related, I could add a link to the persistence layer documentation Magento has but above there's not much listed in documentation

https://devdocs.magento.com/guides/v2.4/architecture/archi_perspectives/persist_layer.html